xml/tuning_memory.xml

```diff
@@ -477,7 +477,7 @@
   </sect2>
 
   <sect2 xml:id="cha-tuning-memory-vm-writetiming" os="sles;sled">
-   <title>Timing Differences of I/O Writes between &sle; 12 and &sle; 11</title>
+   <title>Timing differences of i/o writes between &sle; 12 and &sle; 11</title>
```

Comment:
I/O should not be changed IMHO as it is kind of an abbreviation.

Reply:
This one is going to be hard to fix because I defined `/` as word-split character, meaning I/O is treated as `I/` + `O`. I could special-case this particular one if this is a bigger issue.